Modules/DialogManagement/src/Game.DialogManagement.Domain/Game.DialogManagement.Domain.csproj

```diff
@@ -9,4 +9,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\..\..\Shared\Game.Shared.Core\Game.Shared.Core.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="StackExchange.Redis" Version="2.7.4" />
```

Comment:
Wydaje mi się, że nie korzystasz z tej zależności w domenie, ale to nie powoduje żadnego problemu.

Reply:
ups, dodałem w złym miejscu 😛